src/components/CategoryPage/CategoryPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { GoodCategory } from 'components/GoodCategory';
+import { Row } from 'antd';
+import { Spinner } from 'components/Spinner';
+import { ErrorAlert } from 'components/ErrorAlert';
+import { useCategoryPage } from './useCategoryPage';
+
+
+
+export const CategoryPage:React.FC=()=>{
+
+  const data=useCategoryPage()
```

Comment:
```suggestion
  const categoryHook=useCategoryPage()
```
или categoryController или хотя бы categories